docker-compose.balena.dev.yml.tmpl

```diff
@@ -34,7 +34,7 @@ services:
       - "/dev/vchiq:/dev/vchiq"
     restart: always
     volumes:
-      - resin-data:/data
+      - anthias-data:/data
```

Comment:
Make sure you have a migration script for this for manual deployments. Also, I don't know if this will work on Balena. You need to make sure that the migration doesn't loose any data for any existing installations.

Reply:
I can revert that in this PR (and create a new one for that matter) to focus on Django view migrations.

Reply:
As per [this Balena doc](https://github.com/balena-io/docs/blob/master/shared/general/persistent-storage.md#using-a-supervisor-with-a-version--v1000), starting Balena Supervisor v10.0, unused volumes are no longer removed from disk.

As for the manual deployments (in Raspberry Pi OS Lite devices), data isn't lost, but we'll have to re-test thoroughly. I'll also modifying the container upgrade script to clean up unused volumes.

Reply:
I tried emulating the update process in my Balena instances, but I faced data loss. I'll just continue this matter on #1917.